Makefile

```diff
@@ -188,22 +189,37 @@ ko-build-transport-local: ## Build local transport container image with `ko`.
 
 # this is used for local testing
 .PHONY: kind-load-image
-kind-load-image:
+kind-load-image: ko-build-controller-manager-local ko-build-transport-local
 	kind load --name ${KIND_HOSTING_CLUSTER} docker-image ${CONTROLLER_MANAGER_IMAGE}
+	kind load --name ${KIND_HOSTING_CLUSTER} docker-image ${TRANSPORT_IMAGE}
 
 # this is used for local testing
 .PHONY: k3d-load-image
-k3d-load-image:
+k3d-load-image: ko-build-controller-manager-local ko-build-transport-local
 	k3d image import ${CONTROLLER_MANAGER_IMAGE} -c ${K3D_HOSTING_CLUSTER}
+	k3d image import ${CONTROLLER_MANAGER_IMAGE} -c ${TRANSPORT_IMAGE}
 
-.PHONY: chart
-chart: manifests kustomize
-	cd config/manager && $(KUSTOMIZE) edit set image controller=$(shell echo ${CONTROLLER_MANAGER_IMAGE} | sed 's/\(:.*\)v/\1/')
+.PHONY: core-chart
+core-chart: manifests kustomize
 	$(KUSTOMIZE) build config/default \
-		| yq '. | select(.kind == "ClusterRole*").metadata.name |= "{{.Values.ControlPlaneName}}-" + .' \
-		| yq '. | select(.kind == "ClusterRoleBinding").roleRef.name |= "{{.Values.ControlPlaneName}}-" + .' \
-		> chart/templates/controller-manager.yaml
+		| yq '. | select(.kind == "ClusterRole*").metadata.name |= "{{\"{{.ControlPlaneName}}\"}}-" + .' \
+		| yq '. | select(.kind == "ClusterRoleBinding").roleRef.name |= "{{\"{{.ControlPlaneName}}\"}}-" + .' \
+		| yq '[.]' \
+		| sed -e 's/^/  /' \
+		> /tmp/kubestellar-controller-manager.yaml
 	git checkout -- config/manager/kustomization.yaml
```

Comment:
Why is this line still here?

Reply:
The `git checkou`?
`make core-chart` is still blaming that file for the moment.
We will remove it when the kustomization is removed.

Reply:
Yes, that `git checkout ...`.
I do not understand what you mean by "`make core-chart` is still _blaming_ that file". As far as I can tell, deleting this `git checkout` command will make no difference (besides simplifying the recipe). I tested with that deletion, and found that `make core-chart` does not modify that file.

Reply:
This line has been removed.